tokens.json

```diff
@@ -439,5 +439,15 @@
     "twitter": "https://twitter.com/caodotcash/",
     "telegram": "https://t.me/caocash",
     "desc": "Gives BCH to im_uname."
+  },
+    "DAIQUIRI": {
+    "name": "Tropical Finance Token",
+    "address": "0x24d8d5Cbc14FA6A740c3375733f0287188F8dF3b",
+    "symbol": "DAIQUIRI",
+    "image": "https://tropical.finance/daiquiri.png",
+    "websites": ["https://app.tropical.finance/"],
+    "twitter": "https://twitter.com/tropicalfi_/",
+    "telegram": "https://t.me/tropicalfinance",
+    "desc": "Tropical Finance is a state of the art DEX and yield farming application on SmartBCH Chain with unique features"
```

Comment:
- Which unique features does DAIQUIRI have?
- Have you released an SDK?

Reply:
Unique features:
-Antibot
-Antiwhale
-Dynamic emission rate
-Developers and team don’t hold ANY token 
-As you can see in our roadmap(https://tropical.finance/#roadmap) we will build also a Launchpad, Bridge, NFT marketplace and staking in which DAIQUIRI will be also used as option for payments

-Yes I forked/pull request the tokens.json file above, DAIQUIRI is listed in our dex (dex.tropical.finance); if any other kind of sdk is needed please feel free to tell us, we’ll make it